### PR TITLE
feat/merged weight copying

### PR DESCRIPTION
**Pull request by Supremesource**
# Pull Request Checklist

Before submitting this PR, please make sure:

- [ ] You have run `cargo clippy` and addressed any warnings
- [ ] You have added appropriate tests (if applicable)
- [ ] You have updated the documentation (if applicable)
- [ ] You have reviewed your own code
- [ ] You have updated changelog (if applicable)

## Description

Please provide a brief description of the changes in this PR.

## Related Issues

Please link any related issues here

### CODE REVIEW

File: .ropeproject/autoimport.db
Comment (saiintbrisson) [inline .ropeproject/autoimport.db:1]: im guessing this one was commited by mistake as well?

File: .vscode/settings.json
Comment (saiintbrisson) [inline .vscode/settings.json:1]: also revert this one

File: Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ RUN if [ -n "$AWS_ACCESS_KEY_ID" ]; then \
         ./sccache --start-server && \
         export RUSTC_WRAPPER="/app/sccache"; \
     fi && \
-    cargo build -p node-subspace --release --locked
+    cargo build -p node-subspace --release --features testnet --locked
```
Comment (saiintbrisson) [inline Dockerfile:37]: oops

Comment (Supremesource) [reply Dockerfile]: solved

File: pallets/offworker/src/process.rs [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+impl<T: Config> Pallet<T> {
+    pub fn process_subnets(
+        subnets: Vec<u16>,
+        acc_id: T::AccountId,
+        current_block: u64,
+    ) -> Vec<u16> {
+        let mut deregistered_subnets = Vec::new();
+
+        subnets.into_iter().for_each(|subnet_id| {
```
Comment (saiintbrisson) [inline pallets/offworker/src/process.rs:11]: for

**Pull request merged**